cypress/e2e/list-management/list-creation.feature

```diff
@@ -6,5 +6,6 @@ Feature: List creation
         When I click the link "Create New List"
         And I click the "Lawyers" radio button
         And I choose the country "France"
+        And I enter "smoke@cautionyourblast.com" in the "publisher" input
```

Comment:
testIds should be the last resort https://testing-library.com/docs/queries/about/#priority. We should be selecting with an accessible selector first. If you are able to select with an accessible selector, then it's more likely someone using AT software would be able to select it. 

I've already created a step that you can use which selects by label. `lists/cypress/support/step_definitions/common/i_enter_string_for_string.js`

```
When("I enter {string} for {string}", (value, name) => {
  cy.findByLabelText(name).type(value);
});
```